azure-pipelines.yml

```diff
@@ -60,6 +60,15 @@ stages:
       inputs:
         forceReinstallCredentialProvider: true
 
+    - template: jenkins-codesign/signType.v1.yml@templates
+
+    - task: MicroBuildSigningPlugin@2
+      inputs:
+        signType: '$(Xamarin.SignType)'
+        feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+      env:
+        TeamName: '$(TeamName)'
```

Comment:
What's `TeamName` meant to be set to? It's not defined and so in this case given the single quotes will literally resolve to $(TeamName)

Reply:
Also recommend placing double quotes around the variables used here so the pipeline fails if the variables can't be resolved

Reply:
Luis recommended using `VSCX Platform Tools` for the team name

Reply:
Fixed in 3daf080